app/src/main/java/com/hamdan/acceleromatorminiproject/view/Navigation.kt

```diff
@@ -0,0 +1,6 @@
+package com.hamdan.acceleromatorminiproject.view
+
+object Navigation {
+    const val MAIN_PAGE = "main page"
+    const val HIGH_SCORE = "high score"
```

Comment:
Shouldn't these be capital snake case?

Reply:
ah that makes more sense, I had forgotten what format the navigation follows

updated to use capital snake_case